README.md

```diff
@@ -33,7 +33,26 @@ export KUBE_TEST_REPO_LIST=$(pwd)/repo_list
 
 ### e2e.test
 
-> TODO
+Make sure you have a working [Kubernetes development environment](https://github.com/kubernetes/kubernetes).
+```bash
+go get -d k8s.io/kubernetes
+cd $GOPATH/src/k8s.io/kubernetes
+./build/run.sh make WHAT=test/e2e/e2e.test
+```
+
+If building on Mac or Windows, you can specify `KUBE_BUILD_PLATFORMS`.
+
+For Windows
+``` bash
```

Comment:
minor nit - I think the space between the backticks and bash is breaking color highlighting

Reply:
Just deleted the spaces! Thanks for the catch.